esx_service/vmdk_ops_test.py

```diff
@@ -1039,6 +1012,55 @@ def test_vmdkops_on_tenant_vm(self):
         error_info = vmdk_ops.executeRequest(vm1_uuid, self.vm1_name, self.vm1_config_path, auth.CMD_REMOVE, self.tenant1_vol1_name, opts)
         self.assertEqual(None, error_info)
 
+    @unittest.skipIf(not vsan_info.get_vsan_datastore(),
+                    "VSAN is not found - skipping vsan_info tests")
+    def testPolicyUpdateBackup(self):
```

Comment:
The proposed code in this PR hits in two ways. One through vsan update policy flow, for which following the test case is added. Second is the policy ls admin command which I have described in manual testing in description. Will add a unit test for this as well.